apps/base/models.py

```diff
@@ -14,3 +19,28 @@ class TimeStampedModel(models.Model):
     class Meta:
         abstract = True
         app_label = 'base'
+
+
+"""
```

Comment:
Also docstring should be updated. Here `django signals function` is a bit confusing. Can we find something else for it?